src/DependencyInjection/ContaoCoreExtension.php

```diff
@@ -45,6 +46,17 @@ public function getAlias()
     /**
      * {@inheritdoc}
      */
+    public function getConfiguration(array $config, ContainerBuilder $container)
+    {
+        // Will add the resource to the container
+        parent::getConfiguration($config, $container);
+
+        return new Configuration($container->getParameter('kernel.debug'));
```

Comment:
To pass kernel parameters to the configuration (see http://symfony.com/doc/current/cookbook/configuration/using_parameters_in_dic.html)